packages/webapp/src/components/PlantingLocation/index.js

```diff
@@ -0,0 +1,89 @@
+import styles from './styles.module.scss';
+import React from 'react';
+import PropTypes from 'prop-types';
+import PageTitle from '../PageTitle/v2';
+import Button from '../Form/Button';
+import ProgressBar from '../ProgressBar';
+import LocationPicker from '../LocationPicker';
+import { useTranslation } from 'react-i18next';
+import { useForm } from 'react-hook-form';
+import Form from '../Form';
+
+export default function PurePlantingLocation({
```

Comment:
A couple comments on this file:
- recall we are no using this page as a form. with that being the case, we should use `<Layout/>` instead of `<Form/>`
- since there is no form to register, we can remove the use of react hook form in this component.
- make indentation consistent